docs/src/architecture/mapper.md

```diff
@@ -1,9 +1,86 @@
-# The mapper
+# The tedge-mapper
 
-Or more appropriately the mappers,
-because there is a specific mapper for each cloud provider,
-a Cumulocity mapper, an Azure mapper.
-
-The purpose of these mappers is to translate
-messages written using the cloud-agnostic [Thin-Edge Json format](thin-edge-json.md),
+The tedge-mapper is a key concept to support multiple cloud providers.
+The purpose is to translate
+messages written using the cloud-agnostic [Thin Edge JSON format](thin-edge-json.md),
 into cloud-specific messages.
+
+tedge-mapper is composed of multiple cloud-specific mappers, such as Cumulocity mapper and Azure mapper.
+Each mapper is responsible for its dedicated cloud.
+> Note: The tedge-mapper contains the Cumulocity mapper only currently.
```

Comment:
I would prefer not to mention that explicitly here. We will have it soon.

Reply:
We need to say how to run these different mappers.

These specific mappers are launched by the respective `tedge connect` command.  For instance, `tedge connect c8y` establishes a bridge to Cumulocity and launches a Cumulocity mapper that translates the messages in the background.